lux/vislib/altair/AltairRenderer.py

```diff
@@ -51,10 +51,9 @@ def create_vis(self, vis, standalone=True):
         """
         # Lazy Evaluation for 2D Binning
         if vis.mark == "scatter" and vis._postbin:
-            vis._mark = "heatmap"
-            from lux.executor.PandasExecutor import PandasExecutor
-
-            PandasExecutor.execute_2D_binning(vis)
+            if lux.config.executor.name == "PandasExecutor":
```

Comment:
It looks like the SQLExecutor supports 2D binning for heatmaps, why do we only do this for PandasExecutor?